contributingGuides/NAVIGATION.md

```diff
@@ -55,11 +65,11 @@ export const ROUTES = {
 
 ```
 
-2. Add `Settings_Workspaces` page to proper RHP flow in `linkingConfig.ts`: https://github.com/Expensify/App/blob/3531af22dcadaa94ed11eccf370517dca0b8c305/src/libs/Navigation/linkingConfig.js#L40-L42
+3. Add `Settings_Workspaces` page to proper RHP flow in `linkingConfig.ts`: https://github.com/software-mansion-labs/expensify-app-fork/blob/fbc11ca729ffa4676fb3bc8cd110ac3890debff6/src/libs/Navigation/linkingConfig.ts#L47-L50
```

Comment:
In a followup PR after the merge we should consider directing to `Expensify/App` repo again.

Reply:
It can be done in this PR as the last think, can't it? Using the top commit SHA1

Reply:
Okey, maybe not the _top_ one, as that's a cycle =P Just the last commit modifying `linkingConfig.ts`